expat/xmlwf/xmlwf.c

```diff
@@ -1051,6 +1051,10 @@ tmain(int argc, XML_Char **argv) {
       }
       outName = (XML_Char *)malloc((tcslen(outputDir) + tcslen(file) + 2)
                                    * sizeof(XML_Char));
+      if (! outName) {
+        tperror(T("Could not allocate memory"));
+        exit(1);
+      }                                   
```

Comment:
I think there is some trailing whitespace on this line that makes clang-format reject this change at https://travis-ci.org/github/libexpat/libexpat/jobs/700904054#L1919 . Could you have a look?

Reply:
Now I removed some whitespace.